src/pages/NewChairConsole.js

```diff
@@ -1,40 +1,426 @@
+import { DataUseTranslation } from '../libs/dataUseTranslation';
+import {Notifications, formatDate} from '../libs/utils';
+import { Theme } from '../libs/theme';
 
-class NewChairConsole extends Component {
-
-  render() {
-    const titleStyle = {
-      color: '#1F3B50',
+const styles = {
+  PAGE: {
+    width: "90%",
+    margin: "0 auto"
+  },
+  TITLE: {
+    fontFamily: "Montserrat",
+    fontWeight: Theme.font.weight.semibold,
+    fontSize: Theme.font.size.title,
+  },
+  SMALL: {
+    fontFamily: 'Montserrat',
+    fontWeight: Theme.font.weight.regular,
+    fontSize: Theme.font.size.small
+  },
+  HEADER_IMG: {
+    width: '60px',
+    height: '60px',
+  },
+  HEADER_CONTAINER: {
+    display: 'flex',
+    flexDirection: "column"
+  },
+  ICON_CONTAINER: {
+    flexBasis: '76px',
+    height: '60px',
+    paddingRight: '16px'
+  },
+  RIGHT_HEADER_SECTION: {
+    display: 'flex',
+    alignItems: 'flex-end'
+  },
+  LEFT_HEADER_SECTION: {
+    display: 'flex',
+    flexDirection: 'row',
+    paddingTop: "3rem"
+  },
+  TABLE: {
+    CONTAINER: {
+      margin: "3rem auto"
+    },
+    HEADER_ROW: {
+      fontFamily: "Montserrat",
+      fontSize: "14px",
+      color: "#00243C",
+      fontWeight: Theme.font.weight.medium,
+      backgroundColor: "#f3f6f7",
+      display: "flex",
+      justifyContent: "center",
+      height: "51px"
+    },
+    RECORD_ROW: {
       fontFamily: 'Montserrat',
-      fontSize: '28px',
-      fontWeight: 600
-    };
-
-    const descriptionStyle = {
-      color: '#1F3B50',
+      fontWeight: Theme.font.weight.regular,
+      fontSize: "14px",
+      display: "flex",
+      justifyContent: "center",
+      height: "48px",
+    },
+    RECORD_TEXT: {
+      color: "#00243C"
+    },
+    DAR_TEXT_HOVER: {
+      cursor: 'pointer',
+      color: '#0cabc5d9'
+    },
+    DATA_REQUEST_TEXT: {
+      color: "#00609F",
+      fontWeight: Theme.font.weight.semibold
+    },
+    //NOTE: play around with the cell measurements
+    TITLE_CELL: {
+      width: "18%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+      margin: "0 2%"
+    },
+    DATA_ID_CELL: {
+      width: "10%",
+      margin: "0 2%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    },
+    SUBMISSION_DATE_CELL: {
+      width: "10%",
+      margin: "0 2%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    },
+    DAC_CELL: {
+      width: "10%",
+      margin: "0 2%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    },
+    ELECTION_STATUS_CELL: {
+      width: "10%",
+      margin: "0 2%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    },
+    ELECTION_ACTIONS_CELL: {
+      width: "23%",
+      margin: "0 2%",
+      display: "flex",
+      justifyContent: "left",
+      alignItems: "center",
+    }
```

Comment:
There's a lot of duplicate code in the cells that could be in a single `CELL` style, then specific cells can merge in their own customized properties.

Reply:
Merge in itself is a relatively expensive operation (lodash effectively iterates over keys and applies them to the next argument in the method, depth first assignment). So when you're dealing with multiple cells across multiple rows it could affect performance. Assign is only one level deep, so there's some issues with nested attributes.